### PR TITLE
Add Graal 1.0.0-rc11

### PR DESCRIPTION
**Pull request by ilopmar**

### CODE REVIEW

**Comment by ilopmar** [comment]
I'm not really sure about the version name. If I run `sdk list java` I see that Graal versions have been renamed to `1.0.0-rc-10-grl`. But the old name is kept in the file: https://github.com/sdkman/sdkman-db-migrations/blob/master/src/main/scala/io/sdkman/changelogs/JavaMigrations.scala#L313

**Comment by marc0der** [comment]
Let's go for the version scheme that we currently have on sdkman. If you can update that I'll merge.

**Comment by ilopmar** [comment]
Done!

File: src/main/scala/io/sdkman/changelogs/JavaMigrations.scala [hunk 1]
```diff
@@ -343,5 +343,21 @@ class JavaMigrations {
       .insert()
   }
 
+  @ChangeSet(order = "069", id = "069-add_graalvm_1_0_0_rc_11", author = "ilopmar")
+  def migrate066(implicit db: MongoDatabase) = {
```
Comment (wololock) [inline src/main/scala/io/sdkman/changelogs/JavaMigrations.scala]: The method with this name already exists in line 309

**Comment by marc0der** [comment]
Looks like the build is failing due to a clashing migration number:
https://travis-ci.org/sdkman/sdkman-db-migrations/builds/479890675

**Comment by ilopmar** [comment]
Oh sh*t, I'm so sorry... @marc0der will you push the fix for this?

**Comment by marc0der** [comment]
:laughing: no problem

**Comment by ilopmar** [comment]
:pray:

**Pull request merged**